package/lib/src/beam_interceptor.dart

```diff
@@ -0,0 +1,45 @@
+import 'package:beamer/beamer.dart';
+import 'package:flutter/widgets.dart';
+
+/// IDK LOL
```

Comment:
the name is good, we don't need this comment anymore :smiley:

Reply:
Sorry, my intrusive thought took over ;-)

Reply:
✅